coresimd/arm/cmsis.rs

```diff
@@ -0,0 +1,318 @@
+#![allow(non_snake_case)]
+
+/// Extracted from [CMSIS 5]'s CMSIS/Core/Include/cmsis_gcc.h
+///
+/// [CMSIS 5]: https://github.com/ARM-software/CMSIS_5
```

Comment:
Should this be `//!` instead?

Reply:
I would also prefer this to be a top-module documentation string (using `//!`).